_posts/2019-08-28-ggplot.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 'GGPLOT Blocks'
```

Comment:
Do you want to keep the names `ggplot` and `dplyr` (which won't make any sense at all to our intended audience, who haven't seen R)? Alternatives would be `plotting` and... um... OK, I don't know what a good name for the data manipulation blocks would be.